packages/create-sitecore-jss/src/templates/angular/.env

```diff
@@ -17,6 +17,9 @@ SITECORE_API_HOST=
 # the resolved Sitecore API hostname + the `graphQLEndpointPath` defined in your `package.json`.
 GRAPH_QL_ENDPOINT=
 
+# Your jss app name. Overrides 'config.appName' defined in a package.json

```

Comment:
We should mention that this is used as the _site name_ in the comment. Maybe something like
```suggestion
# Your JSS app name (also used as the default site name). Overrides 'config.appName' defined in a package.json

```